src/com/raghav/javabasics/DisplayArrayNumbersAreDivisible.java

```diff
@@ -0,0 +1,29 @@
+package com.raghav.javabasics;
+import java.util.Arrays;
+
+//Given an array of numbers, write a program to identify all numbers divisible by 2,
+// divisible by 3, divisible by 5, divisible by 7, divisible by 5 and divisible by 7.
+
+public class DisplayArrayNumbersAreDivisible {
+    public static void main(String[] args) {
+
+        DisplayArrayNumbersAreDivisible result = new DisplayArrayNumbersAreDivisible();
+        result.divisibilityChecker(new int[]{1,2,3,4,5,6,7,8,9,35});
+        result.divisibilityChecker(new int[]{});
```

Comment:
What about 0 and negative numbers?

Reply:
output:
Array Numbers Divisible by 2: [2, 4, 6, 8, 0, 0, 0, 0, 0, 0]
Array Numbers Divisible by 3: [3, 6, 9, 0, 0, 0, 0, 0, 0, 0]
Array Numbers Divisible by 5: [5, 35, 0, 0, 0, 0, 0, 0, 0, 0]
Array Numbers Divisible by 7: [7, 35, 0, 0, 0, 0, 0, 0, 0, 0]
Array Numbers Divisible by 35: [35, 0, 0, 0, 0, 0, 0, 0, 0, 0]
Array Numbers Divisible by 2: []
Array Numbers Divisible by 3: []
Array Numbers Divisible by 5: []
Array Numbers Divisible by 7: []
Array Numbers Divisible by 35: []

Reply:
Negative Numbers?

Reply:
Added in the latest commit